ush/templates/parm/metplus/GridStat_REFC_prob_NMEP_tk.conf

```diff
@@ -0,0 +1,337 @@
+# Ensemble probabilistic GridStat METplus Configuration
```

Comment:
This file is the same as GridStat_REFC_prob.conf, right?

Reply:
It should be.  Let me double check

Reply:
It's not.  It's the test file that we used.  I deleted it